script/external/DeployGovernor_v1.s.sol

```diff
@@ -43,14 +44,20 @@ contract DeployGovernor_v1 is Script {
             "DeployOrchestratorFactory_v1: Missing env variable: team multisig"
         );
 
+        require(
+            feeManager != address(0),
+            "DeployOrchestratorFactory_v1: Missing env variable: feeManager"
+        );
+
         // Deploy the Governor_v1.
-        return run(communityMultisig, teamMultisig, timelockPeriod);
+        return run(communityMultisig, teamMultisig, timelockPeriod, feeManager);
     }
 
     function run(
         address communityMultisig,
         address teamMultisig,
-        uint timelockPeriod
+        uint timelockPeriod,
+        address initialFeeManager
```

Comment:
Why do we actually get the variable from our .env as well as from the function call? I'm seeing that this happens elsewhere also

Reply:
This is to test the scripts in the Make pre-commit
We basically mock the other addresses that way

Reply:
Ahh okay, makes sense